lib/MilkCheck/Engine/ServiceGroup.py

```diff
@@ -211,9 +211,9 @@ def remove_inter_dep(self, dep_name):
         if not self.has_subservice(dep_name):
             raise ServiceNotFoundError()
         else:
-            for dep in self._subservices[dep_name].parents.values():
+            for dep in self._subservices[dep_name].parents.copy().values():
```

Comment:
same here and below

Reply:
Done.

I did the same for ServiceManager also.